src/openforms/formio/typing/base.py

```diff
@@ -53,7 +57,7 @@ class PrefillConfiguration(TypedDict):
     identifierRole: Literal["main", "authorised_person"]
 
 
-class Component(TypedDict, total=False):
+class Component(TypedDict):
```

Comment:
I think it would make more sense to make it `total=False` and use `Required` for the couple required attributes?

Reply:
discussed in office, leaving this one as it is :)